### ISSUE TITLE
Rename and convert `calspec` files

### ISSUE THREAD

**Issue by kelle** [pyspextool/pyspextool#155]
write some functions which 
- copy and rename "calspec" files to `instrument_mode_radec_object_yyyymmdd_pyspextool.fits`
- convert to Spectrum1D fits files and rename to `instrument_mode_radec_object_yyyymmdd.fits`

Happy to chat about the filename convention here.

**Comment by michaelcushing** [comment]
When you say "calspec" do you mean the files like ShortXD_wavecalinfo.fits?

**Comment by kelle** [comment]
It would also be good if this script uploaded the "done" files to a "staging" google drive folder for ingest into SIMPLE.

**Comment by michaelcushing** [comment]
I think we have to keep the core code separate from all the other things we want to do.  And so perhaps we write something that is run while we are reducing all the data to do this kind of thing?

**Comment by kelle** [comment]
I think this can be part of Adam's batch module? Isn't that already a little "extra" beyond the fundamental of pyspextool?

But also, what do you think about filenames? I think we should somehow distinguish between Spextool FITS vs Spectrum1d FITS.

**Comment by michaelcushing** [comment]
I agree we need to distinguish.  I am just not sure we need the entire string of things.  Having it as an option though is a great idea and I will get that into the code when I go back around and for a second pass on everything.

**Comment by aburgasser** [comment]
Deferring this until Spectrum1D issues are addressed
